volume/drivers/api.go

```diff
@@ -4,11 +4,13 @@ package volumedrivers
 
 import "github.com/docker/docker/volume"
 
+// NewVolumeDriver returns a driver has the given name mapped on the given client.
 func NewVolumeDriver(name string, c client) volume.Driver {
 	proxy := &volumeDriverProxy{c}
 	return &volumeDriverAdapter{name, proxy}
 }
 
+// VolumeDriver manages volumes from end to end.
```

Comment:
Looking at volumes and plugins for different reasons and I think I understand this a bit more. How is:
`VolumeDriver defines the available functions that volume plugins must implement.`